tutorials/plot_point_spread.py

```diff
@@ -0,0 +1,178 @@
+"""
+.. _point_spread:
+
+Corrupt known signal with point spread
+======================================
+
+The aim of this tutorial is to demonstrate how to put a known signal at a
+desired location(s) in a :class:`mne.SourceEstimate` and then corrupt the
+signal with point-spread by applying a forward and inverse solution.
+"""
+
+import os.path as op
+
+import numpy as np
+
+import mne
+from mne.datasets import sample
+
+from mne.minimum_norm import read_inverse_operator, apply_inverse
+from mne.simulation import simulate_stc, simulate_evoked
+
+###############################################################################
+# First, we set some parameters.
+
+seed = 42
+
+# parameters for inverse method
+method = 'sLORETA'
+snr = 3.
+lambda2 = 1.0 / snr ** 2
+
+# signal simulation parameters
+# do not add extra noise to the known signals
+evoked_snr = np.inf
+T = 100
+times = np.linspace(0, 1, T)
+dt = times[1] - times[0]
+
+# Paths to MEG data
+data_path = sample.data_path()
+subjects_dir = op.join(data_path, 'subjects')
+fname_fwd = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis-meg-oct-6-fwd.fif')
+fname_inv = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis-meg-oct-6-meg-fixed-inv.fif')
+
+fname_evoked = op.join(data_path, 'MEG', 'sample',
+                       'sample_audvis-ave.fif')
+
+###############################################################################
+# Load the MEG data
+# -----------------
+
+fwd = mne.read_forward_solution(fname_fwd, force_fixed=True,
+                                surf_ori=True)
+fwd['info']['bads'] = []
+inv_op = read_inverse_operator(fname_inv)
+
+raw = mne.io.RawFIF(op.join(data_path, 'MEG', 'sample',
+                            'sample_audvis_raw.fif'))
+events = mne.find_events(raw)
+event_id = {'Auditory/Left': 1, 'Auditory/Right': 2}
+epochs = mne.Epochs(raw, events, event_id, baseline=(None, 0), preload=True)
+epochs.info['bads'] = []
+evoked = epochs.average()
+
+labels = mne.read_labels_from_annot('sample', subjects_dir=subjects_dir)
+label_names = [l.name for l in labels]
+n_labels = len(labels)
+
+###############################################################################
+# Estimate the background noise covariance from the baseline period
+# -----------------------------------------------------------------
+
+cov = mne.compute_covariance(epochs, tmin=None, tmax=0.)
+
+###############################################################################
+# Generate sinusoids in two spatially distant labels
+# --------------------------------------------------
+
+# The known signal is all zero-s off of the two labels of interest
+signal = np.zeros((n_labels, T))
+idx = label_names.index('inferiorparietal-lh')
+signal[idx, :] = 1e-7 * np.sin(5 * 2 * np.pi * times)
+idx = label_names.index('rostralmiddlefrontal-lh')
```

Comment:
I would use `rostralmiddlefrontal-rh`, unless you're trying to show that it doesn't spread across the hemispheres (if so, mention it in the comments)